test/gtest/common/test_helpers.cc

```diff
@@ -606,8 +606,16 @@ uint16_t get_port() {
     return port;
 }
 
-void *mmap_fixed_address() {
-    return (void*)0xff0000000;
+void *mmap_fixed_address(size_t length) {
+    void *ptr = mmap(NULL, length, PROT_READ | PROT_WRITE,
+                     MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
+    if (ptr == MAP_FAILED) {
+        return NULL;
```

Comment:
You can use `nullptr` in C++ code.
```suggestion
        return nullptr;
```